tests/Webapi/Dom/Webapi__Dom__Event__test.re

```diff
@@ -14,7 +14,10 @@ let _ = target(event);
 let _ = timeStamp(event);
 let _ = type_(event);
 let _ = isTrusted(event);
+let _ = returnValue(event);
 
 preventDefault(event);
 stopImmediatePropagation(event);
 stopPropagation(event);
+setReturnValue(event, "Test");
+deleteReturnValue(event);
```

Comment:
Can you commit the JS artifacts as well so we can ensure this generates correct code?